src/main/java/com/patchworkmc/event/EventSubscriber.java

```diff
@@ -0,0 +1,59 @@
+package com.patchworkmc.event;
+
+import java.util.Objects;
+
+/**
+ * Object to hold information about a class that has at least one event subscriber.
+ */
+public class EventSubscriber {
+	private final String className;
+	private final boolean isInterface;
+	private final boolean hasInstanceSubscriber;
+	private final boolean hasStaticSubscriber;
+
+	public EventSubscriber(String className, boolean isInterface, boolean hasInstanceSubscriber, boolean hasStaticSubscriber) {
```

Comment:
`hasInstanceSubscribers`, maybe?

Reply:
No, because then that would imply that there's always more than one

Reply:
Not necessarily, though?